### PR TITLE
Type identifier in preview compact bubble.

### PR DESCRIPTION
**Pull request by reddyashish**
### Purpose

Task: https://jira.autodesk.com/browse/DYN-5053

Add Type identifiers in preview compact bubble. Similar to preview bubble or watch node.

type identifier

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes
Type identifier in preview compact bubble.

### Reviewers
@QilongTang @zeusongit

### CODE REVIEW

**Comment by reddyashish** [comment]
@QilongTang I did not need to call the type overloading methods in compact preview bubble separately, like I initially thought. Getting the type info using the Data property inside MirrorData.

File: src/DynamoCoreWpf/ViewModels/Preview/CompactBubbleViewModel.cs [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
     /// </summary>
     public class CompactBubbleViewModel : ViewModelBase
     {
         #region Properties
-
+        internal const string doubleType = "double";
+        internal const string boolType = "bool";
+        internal const string intType = "int";
+        internal const string objectType = "object";
+        internal const string stringType = "string";
```
Comment (QilongTang) [inline src/DynamoCoreWpf/ViewModels/Preview/CompactBubbleViewModel.cs:18]: Are these defined somewhere already so we can make them static to reuse?

**Review by QilongTang — APPROVED** [review]
One question then LGTM

File: src/DynamoCoreWpf/ViewModels/Preview/CompactBubbleViewModel.cs (hunk 1 above)
Comment (mjkkirschner) [reply src/DynamoCoreWpf/ViewModels/Preview/CompactBubbleViewModel.cs:18]: can't you use `nameof`?

Comment (reddyashish) [reply src/DynamoCoreWpf/ViewModels/Preview/CompactBubbleViewModel.cs]: Yes, there constants were used in WatchViewModel. Removing it from WatchViewModel and CompactBubbleViewModel and using nameof property.

Comment (QilongTang) [reply src/DynamoCoreWpf/ViewModels/Preview/CompactBubbleViewModel.cs]: Thanks! Looks good

**Pull request merged**